src/collections/_documentation/error-reporting/configuration/drain-example/rust.md

```diff
@@ -2,11 +2,14 @@ When the Rust SDK initializes a guard is returned from the `init` function.  The
 will automatically wait `shutdown_timeout` seconds.  This means you just need to hold on to
 the guard and make sure it disposes on shutdown.  Alternatively the client can be closed:
 
-```javascript
+```rust
 use std::time::Duration;
 use sentry::Hub;
 
 if let Some(client) = Hub.current().client() {
     client.close(Some(Duration::from_secs(2)));
 }
 ```
+
+After shutdown the current client cannot be used any more so make sure to only
+do that right before you shut down the application.
```

Comment:
This is moved here from the global context, with one important edit: "the client" => "the **current** client".